tests/benchmark/benchmark_between_releases/Dockerfile.in

```diff
@@ -0,0 +1,24 @@
+FROM ${FULL_BASE_IMAGE}
```

Comment:
Do you think we could simplify this by using the Dockerfile from tests/benchmark/ for both version-to-version and release-to-release benchmarking?